Packs/FiltersAndTransformers/Scripts/RemoveMatches/RemoveMatches.yml

```
@@ -0,0 +1,47 @@
+args:
+- default: true
+  description: The value on which the transformer is applied.
+  isArray: true
+  name: value
+- auto: PREDEFINED
+  defaultValue: "Yes"
+  description: Whether to ignore the case of the item for which you are searching. Default is "Yes".
+  name: ignorecase
```

```suggestion
  name: ignore_case
```
Please update accordingly the python file and the readme.